packages/saudi-arabia/saudi-arabia.svg

```diff
@@ -0,0 +1,59 @@
+     title="makkah"
```

Comment:
Could you use the same names as in the `README.md` file (Riyadh, Makkah...) please?